src/tools/dev/scripts/bv_support/bv_mesagl.sh

```diff
@@ -262,6 +262,36 @@ EOF
         return 1
     fi
 
+    #
+    # Patch to address VTK texture buffer error.
+    #
+    patch -p0 << \EOF
+diff -c src/gallium/drivers/llvmpipe/lp_screen.c.orig src/gallium/drivers/llvmpipe/lp_screen.c
+*** src/gallium/drivers/llvmpipe/lp_screen.c.orig        Fri Dec 15 14:33:53 PST 2023
+--- src/gallium/drivers/llvmpipe/lp_screen.c     Fri Dec 15 14:33:53 PST 2023
+***************
+*** 236,242 ****
+     case PIPE_CAP_TEXTURE_BUFFER_OBJECTS:
+        return 1;
+     case PIPE_CAP_MAX_TEXTURE_BUFFER_SIZE:
+!       return 65536;
+     case PIPE_CAP_TEXTURE_BUFFER_OFFSET_ALIGNMENT:
+        return 1;
+     case PIPE_CAP_PREFER_BLIT_BASED_TEXTURE_TRANSFER:
+--- 236,242 ----
+     case PIPE_CAP_TEXTURE_BUFFER_OBJECTS:
+        return 1;
+     case PIPE_CAP_MAX_TEXTURE_BUFFER_SIZE:
+!       return 134217728;
```

Comment:
Was this taken from a newer version of Mesa? If not how did you come up with the number?

Reply:
This was taken from here: https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/9750